plaidml/op/op.h

```diff
@@ -337,6 +337,65 @@ inline edsl::Tensor minimum(const edsl::Tensor& X, const edsl::Tensor& Y) {
   return details::op("minimum", args).as_tensor();
 }
 
+class mvn {
+ public:
+  explicit mvn(const plaidml::edsl::Tensor& I)
+      : I_(I),
+        axes_(edsl::None()),
+        normalize_variance_(true),
+        epsilon_(1e-9),
+        across_channels_(true),
+        layout_("NHWC") {}
+
+  mvn& axes(const std::vector<int64_t>& axes) {
+    // negative axes interpreted as in numpy
+    if (!across_channels_ || layout_ != "NHWC") {
```

Comment:
Hmm, the only actual requirement is that `N` and `C` are in the layout, but I don't think a specific fully formed layout is required.

Reply:
Sorry, actually just `C` must exist, and then if `N` exists we skip it.

Reply:
This check is only implementing "And also we could do the same for axes (if layout was set, then throw)." (since "NHWC" is the default value of layout, this throws when layout is set.) Do you want a requirement checking that `C` exists? I currently don't do any such check at all, just skip "N" (and "C" if `across_channels_ == true`)

Reply:
Hmm, maybe we shouldn't have a default for layout then... 

It seems like we want two exclusive groups of options:
- group 1:
  - axes
- group 2:
  - across_channels
  - layout

Reply:
OK, maybe we just say `if (!layout_.empty()) throw`